site/content/blog/2022-07-01-whats-new-in-svelte-july-2022.md

```diff
@@ -0,0 +1,91 @@
+---
+title: "What's new in Svelte: July 2022"
+description: "Faster SSR, Language tools improvements and something special..."
+author: Daniel Sandoval
+authorURL: https://desandoval.net
+---
+
+With [something special](https://twitter.com/SvelteSociety/status/1539284972544180230?s=20&t=3Qoi371NrL9LP6xMOF63ag) dropping today along with faster SSR and a number of new language tools features, there's a lot to cover in this month's newsletter...
```

Comment:
"a number of new language tools features" is kind of vague - maybe swap it for something a bit more specific?
```suggestion
With [something special](https://twitter.com/SvelteSociety/status/1539284972544180230?s=20&t=3Qoi371NrL9LP6xMOF63ag) dropping today along with faster SSR and support for Vitest and Storybook in SvelteKit, there's a lot to cover in this month's newsletter...
```